### DISCUSSION TITLE
Clarity Progress bar with multiple series

### DISCUSSION THREAD

**Discussion by kumar-tadepalli** [vmware/clarity#5979] [Q&A]
Hi Team,

Is it possible to achieve multiple series based progress bar as shown below?
![image](https://user-images.githubusercontent.com/12848838/119082520-e263b280-ba1b-11eb-9df8-69361bb47336.png)

Thanks,
Sravan

**Comment by mathisscott** [comment]
@shravansofts 
This is one of the progress bar variations we would like to investigate when we bring progress bars into the Clarity Core library. We haven't started planning that out yet so I can't give a commitment on time or features.
